packages/algolia/src/index.ts

```diff
@@ -0,0 +1,224 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion, no-else-return */
+import sortBy from 'lodash/sortBy.js';
+import reduce from 'lodash/reduce.js';
+import isString from 'lodash/isString.js';
+import isArray from 'lodash/isArray.js';
+import flatten from 'lodash/flatten.js';
+import each from 'lodash/each.js';
```

Comment:
```suggestion
import { sortBy, reduce, isString, isArray, flatten, each } from 'lodash';
```

Reply:
Does not work in ESM, that's why I've put it this way

Reply:
can you install only functions that don't exist in native js to make bundler smaller, I guess:
```
reduce => Array.reduce
isString => typeof foo === 'string'
isArray => Array.isArray
flatten => Array.flat
each => Array.forEach or for (const [key,value] of Object.entries)
```